src/core_modules/capture-core/hooks/useProgramInfo/index.js

```diff
@@ -1,3 +1,3 @@
 // @flow
-export { useProgram } from './useProgram';
+export { useProgramInfo } from './useProgramInfo';
 export { programTypes } from './programTypes.const';
```

Comment:
I was thinking that at the moment we only have three types in total. 

1. TEType.
2. Event Program
3. Tracker Pogram

I have created [my own constant](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/metaData/helpers/getScopeFromScopeId.js#L6) to describe those. I think it would be better if we can align on this so that we have only one constant describing the types around the app. What you think?

Reply:
Good point 👍 Could have been a different PR, but made a suggestion in the last commit in this one. Please have a look @paschalidi.

Reply:
Yeah looks rad!